test/integration/targets/known_hosts/tasks/main.yml

```diff
@@ -170,6 +170,95 @@
     - 'result.diff.before == result.diff.after'
     - 'known_hosts_v3.stdout == known_hosts_v4.stdout'
 
+# test addition as hashed_host
+
+- name: add a new hashed host
+  known_hosts:
+    name: example.org
+    key: "{{ example_org_rsa_key }}"
+    state: present
+    path: "{{output_dir|expanduser}}/known_hosts"
```

Comment:
`expanduser` should not be called:
- it works because in the CI controller and managed node are the same
- type of `path` parameter is `path`: [expanduser](https://github.com/ansible/ansible/blob/f7d9b827053a4342effab42af244868ffde11ce2/lib/ansible/module_utils/basic.py#L1967) is called on the managed host

Reply:
What should be used instead? I just copied the existing tests…

Reply:
Nothing else is required `path: "{{output_dir}}/known_hosts"` works well.